ete_treematcher.py

```diff
@@ -13,35 +18,33 @@ def populate_args(extract_args_p):
     """
     extract_args = extract_args_p.add_argument_group('TREEMATCHER OPTIONS')
 
-    extract_args.add_argument("--pattern", dest="pattern",
-                              nargs=1,
-                              help="")
-    extract_args.add_argument("--pattern-format", dest="pattern_format",
```

Comment:
No need for pattern format. It is now always forced to be format 1 and quoted names